OpenTap.Plugins.Ssh/SshCommandStep.cs

```diff
@@ -62,32 +72,77 @@ public SshStepBase()
     public class SshCommandStep : SshStepBase
     {
         #region Settings
-        public string Command { get; set; }
+        public string Command { get; set; } = "pwd";
+
+        [Display("Add To Log", Group: "Response", Collapsed: true, Order:0)]
+        public bool AddToLog { get; set; } = true;
+        [Display("Output Response", "Sets if the output of the ssh command should be saved as an output.", Group: "Response", Collapsed: true, Order:0)]
+        public bool OutputResponse { get; set; }
+
+        [Output]
+        [Browsable(true)]
+        [EnabledIf(nameof(OutputResponse), HideIfDisabled = true)]
+        [Display("Response", Group: "Response", Collapsed: true, Order:1)]
+        public string Response { get; private set; }
+
+        [Output]
+        [Browsable(true)]
+        [Display("Exit Code", Group: "Response", Collapsed: true, Order:1)]
+        public int ExitCode { get; private set; }
```

Comment:
Should we have a Display.Description that explains that this sets the Verdict of the step?